wire-ios-request-strategy/Sources/Request Strategies/User/UserProfileRequestStrategyTests.swift

```diff
@@ -201,6 +220,7 @@ class UserProfileRequestStrategyTests: MessagingTestBase {
     // MARK: - Response processing
 
     func testThatUsesLegacyEndpoint_WhenFederatedEndpointIsDisabled() {
+        // TODO: check if this test is right?
```

Comment:
this passes because we're on v0 but I am questioning the veracity of the description of this test.